test/topology/conftest.py

```diff
@@ -225,6 +226,14 @@ async def manager(request, manager_internal, record_property, build_mode):
 def cql(manager):
     yield manager.cql
 
+@pytest.fixture(scope="function")
+async def opensearch():
+    cluster = OpenSearchCluster(logger=logging.getLogger())
+    await cluster.install_and_start()
+    yield cluster
```

Comment:
As a proof that this PR works and sets up an OpenSearch instance properly, the dummy test should attempt to send some request to the instance - something non-destructive, like e.g. list the current indexes or something like that.